javascript/javascript2/week1/countOfDanishLetters.js

```diff
@@ -0,0 +1,36 @@
+/* Find the individual number and the total number of danish letters in a string.
+
+const danishString = ['Jeg har en blå bil'];
+notThisFunctionName(danishString); // returns {total: 1, å: 1}
+
+const danishString2 = ['Blå grød med rød bær'];
+notThisFunctionName(danishString2); // returns {total: 4, æ: 1, ø: 2, å: 1} */
+const danishString = ["Jeg har en blå bil"];
+const danishString2 = ["Blå grød med rød bær"];
+
+function countOfDanishLetters(listOfWords) {
+  // danish letters æ å ø
+  let æCount = 0;
+  let åCount = 0;
+  let øCount = 0;
+  for (let i = 0; i < listOfWords[0].length; i++) {
+    if (listOfWords[0][i] === "æ" || listOfWords[0][i] === "Æ") æCount++;
+    else if (listOfWords[0][i] === "å" || listOfWords[0][i] === "Å") åCount++;
+    else if (listOfWords[0][i] === "ø" || listOfWords[0][i] === "Ø") øCount++;
+  }
+  let totalLetters = æCount + åCount + øCount;
+  console.log(
+    "Total Danish Words:" +
+      totalLetters +
+      " æ: " +
+      æCount +
+      " ø: " +
+      øCount +
+      " å: " +
+      åCount
+  );
+}
```

Comment:
Good job. it works. You can improve this by reducing number of times that for loop executes. 
Now listOfWords[] consists white spaces as well., so if you can remove all spaces and take only letters then your code execution will be fast since you are reducing the loop execution which is happening for spaces also.

Reply:
here listOfWords[] is array, not a string. if it is a string we can remove all the spaces with regular expression. i think it is unnecessary to run a loop over array to remove spaces. Do you have any code snippet to remove spaces from array?

Reply:
There are no comma after each word in array, so its considering it as string. Can you console.log listOfWords[0].length and verify this again?

Reply:
Yes, you are correct. implemented this